src/tests/ipc_tests.rs

```diff
@@ -0,0 +1,274 @@
+#[cfg(test)]
+pub mod ipc_tests {
+    use crate::interface;
+    use crate::safeposix::{cage::*, filesystem, dispatcher::*};
+    use super::super::*;
+    use std::os::unix::fs::PermissionsExt;
+    use std::fs::OpenOptions;
+    use std::time::Instant;
+
+    //#[test]
+    pub fn test_ipc() {
+        // These can't really run until we figure out a better testing system/fsutils
+        //ut_lind_ipc_pipefile();
+        //ut_lind_ipc_pipe();
+        ut_lind_ipc_domain_socket();
+        ut_lind_ipc_socketpair(); 
+    }
+
+
+    pub fn ut_lind_ipc_pipefile() {
+        let byte_chunk: usize = 131072;
+        let num_writes: usize = 8192;
+
+        lindrustinit(0);
+
+        let cage = interface::cagetable_getref(1);
+
+
+        let filefd = cage.open_syscall("test1gb.txt", O_CREAT | O_WRONLY, S_IRWXA);
+
+        let mut buf: Vec<u8> = vec!['A' as u8; byte_chunk];
+        let bufptr = buf.as_mut_ptr();
+
+        for _i in 0..num_writes {
+            cage.write_syscall(filefd, bufptr, byte_chunk);
+        }
+
+        assert_eq!(cage.close_syscall(filefd), 0);
+        assert_eq!(cage.exit_syscall(EXIT_SUCCESS), EXIT_SUCCESS);
+        lindrustfinalize();
+    }
+    pub fn ut_lind_ipc_pipe() {
+
+        let byte_chunk: usize = 131072;
+        let num_writes: usize = 8192;
```

Comment:
```suggestion
        let byte_chunk: usize = 0x20000;
        let num_writes: usize = 0x2000;
```

Reply:
I dont think it really makes sense to change these to hex.